src/py/flwr/server/app.py

```diff
@@ -559,6 +582,23 @@ def _try_setup_node_authentication(
         )
 
 
+def _try_obtain_user_auth_config(args: argparse.Namespace) -> Optional[dict[str, Any]]:
+    if args.user_auth_config is not None:
+        with open(args.user_auth_config, encoding="utf-8") as file:
+            config: dict[str, Any] = yaml.safe_load(file)
+            return config
+    return None
+
+
+def _try_obtain_exec_auth_plugin(config: dict[str, Any]) -> Optional[ExecAuthPlugin]:
+    auth_config: dict[str, Any] = config.get("authentication", {})
+    all_plugins: dict[str, type[ExecAuthPlugin]] = get_exec_auth_plugins()
```

Comment:
Should we wrap this with a `try/except` and, when the `ImportError` is raised, we display a short but informative error message instead of showing the error+traceback?

Reply:
Yes. The line above should also be followed by an error message if the config file doesn't contain the expected field.

Reply:
Thanks guys, I've updated it.